integrations/api_notification_test.go

```diff
@@ -0,0 +1,105 @@
+// Copyright 2019 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package integrations
+
+import (
+	"code.gitea.io/gitea/models"
+	api "code.gitea.io/gitea/modules/structs"
+	"fmt"
+	"net/http"
+	"testing"
```

Comment:
Proper formatting.

Reply:
and there it happend ... :sweat_smile:

Reply:
@adelowo done (sometimes my IDE mess up with this)